i18n/en.yaml

```diff
@@ -161,3 +161,5 @@ pagination:
 memos:
   author: By <a href="{{ .profile }}" target="_blank" rel="noopener">{{ .name }}</a>, a third-party contributor
 reuses: Reuses
+presentations:
+  language: In {{ .language }} language
```

Comment:
```suggestion
  language: In {{ .language }}
```

Reply:
In English, language names have a capital. It should thus read “In French” and not “In french language”.